android/composeui/src/main/java/com/stadiamaps/ferrostar/composeui/views/components/speedlimit/SpeedLimitView.kt

```diff
@@ -0,0 +1,39 @@
+package com.stadiamaps.ferrostar.composeui.views.components.speedlimit
+
+import android.content.Context
+import android.icu.util.ULocale
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import com.stadiamaps.ferrostar.composeui.formatting.MeasurementSpeedFormatter
+import com.stadiamaps.ferrostar.core.measurement.MeasurementSpeed
+import com.stadiamaps.ferrostar.core.measurement.MeasurementSpeedUnit
+
+enum class SignageStyle {
+  MUTCD,
+  ViennaConvention
+}
+
+@Composable
+fun SpeedLimitView(
+    modifier: Modifier = Modifier,
+    speedLimit: MeasurementSpeed,
+    signageStyle: SignageStyle = SignageStyle.ViennaConvention, // TODO: This could be nicer
```

Comment:
Let's remove the default (recursively); discussed on the call.

(Same on iOS)

Reply:
Sorted 👍